docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -33,6 +38,7 @@ The AI assistant requires the following:
 * An account with a third-party generative AI provider that supports function calling. The Observability AI Assistant supports the following providers:
 ** OpenAI `gpt-4`+.
 ** Azure OpenAI Service `gpt-4`(0613) or `gpt-4-32k`(0613) with API version `2023-07-01-preview` or more recent.
+** AWS Bedrock, specifically the Anthropic Claude models.
```

Comment:
Right now we do not support Claude 3: https://github.com/elastic/security-team/issues/8857

Reply:
Added as a restriction. We'll need to make sure the restriction gets removed when support is added.